### PR TITLE
Top module reports UI

### PR DESCRIPTION
**Pull request by skochay330**
Created UI for all top module reports (most used modules, module usage by organization, module usage by job template, module usage by task).

Jira issues: https://issues.redhat.com/browse/AA-754, https://issues.redhat.com/browse/AA-828, https://issues.redhat.com/browse/AA-829, https://issues.redhat.com/browse/AA-830

Parent Jira card: https://issues.redhat.com/browse/AA-546

Screen Shot 2021-10-28 at 2 48 54 PM
Screen Shot 2021-10-28 at 2 49 14 PM
Screen Shot 2021-10-28 at 2 49 33 PM
Screen Shot 2021-10-28 at 2 49 51 PM
Screen Shot 2021-10-28 at 2 50 13 PM

### CODE REVIEW

File: src/Containers/Reports/Shared/schemas/index.ts [hunk 1]
```diff
@@ -25,7 +29,27 @@ const prodReports = [
...
-  prodReports.find(({ slug }) => slug === searchSlug) ?? defaultReport;
+const moduleReports = [
+  mostUsedModules,
+  moduleUsagebyOrganization,
+  moduleUsageByJobTemplate,
+  moduleUsageByTask,
+];
+
+export const getReport = (searchSlug: string): ReportPageParams => {
+  const moduleReportsEnabled = useFeatureFlag(ValidFeatureFlags.moduleReports);
```
Comment (h-kataria) [inline src/Containers/Reports/Shared/schemas/index.ts:40]: move this out to above line 39 then you can delete line 52

Comment (h-kataria) [reply src/Containers/Reports/Shared/schemas/index.ts:40]: ignore my previous comment, i see it is causing the error when using hooks outside the function

**Review by h-kataria — APPROVED** [review]

File: src/Containers/Reports/Shared/schemas/index.ts (hunk 1 above)
Comment (skochay330) [reply src/Containers/Reports/Shared/schemas/index.ts:40]: This change breaks the UI because useFeatureFlag hook cannot be used outside of a function

**Comment by h-kataria** [comment]
created flags to gate these reports
Screen Shot 2021-11-01 at 5 13 51 PM

**Pull request merged**